bot/controllers/roles.js

```diff
@@ -109,11 +113,104 @@ module.exports = () => {
     return false;
   };
 
+  // VerifyFS role
+  const _verifyFs = (message) => {
+    util.log('passed', message.content);
+    const uuid = uuidv4();
+    const msg = messageMiddleware(message);
+
+    if (msg.parsed[0].toLowerCase() === '!verifyfs') {
+      console.log("🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥")
```

Comment:
Not in compliance with the AirBnB Styleguide. Please ensure you are running your unit tests locally to identify oversights.